packages/contentful--create-contentful-app/src/template.ts

```diff
@@ -4,29 +4,102 @@ import degit from 'degit';
 import rimraf from 'rimraf';
 
 import { CLIOptions, ContentfulExample } from './types';
-import { highlight, warn } from './logger';
+import { highlight, success, warn, wrapInBlanks } from './logger';
 import { rmIfExists } from './utils';
+// @ts-expect-error no types available
```

Comment:
Is it possible to use `@types/node-fetch` ?

Reply:
Weirdly when I tried it the first time it didn't work, but know it seems to work 🤔